flapison/resource.py

```diff
@@ -221,9 +219,13 @@ def get(self, *args, **kwargs):
     @check_method_requirements
     def post(self, *args, **kwargs):
         """Create an object"""
-        qs = QSManager(request.args, self.schema)
+
         json_data = self.parse_request()
 
+        self.before_post(args, kwargs, json_data=json_data)
```

Comment:
If you rename this kwarg, you also need to change the function signature of `before_post(args, kwargs, data=None)`